src/navigations/HomeBottomTabs.js

```diff
@@ -165,23 +180,17 @@ function HomeBottomTabs() {
 export default HomeBottomTabs;
 
 const styles = StyleSheet.create({
-  container: {
-    height: '100%',
-    width: '100%'
-  },
-  badge: {
-    height: 7,
-    width: 7,
-    position: 'absolute',
-    bottom: 3,
-    borderRadius: 3.5
-  },
-  center: {
+  iconContainer: {
+    top: dimen.normalizeDimen(5),
     alignItems: 'center',
     justifyContent: 'center'
   },
-  icon: {
-    width: 21,
-    height: 20
+  bottomBarBadge: {
+    fontFamily: fonts.inter[700],
+    fontSize: normalizeFontSize(10)
+  },
+  bottomBarLabel: {
+    fontFamily: fonts.inter[500],
+    fontSize: normalizeFontSize(10)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The styles have been updated to include new styles for `iconContainer`, `bottomBarBadge`, and `bottomBarLabel`. These changes are likely related to the updates in the component's structure and appearance.
```diff
+ iconContainer: {
+   top: dimen.normalizeDimen(5),
+   alignItems: 'center',
+   justifyContent: 'center'
+ },
+ bottomBarBadge: {
+   fontFamily: fonts.inter[700],
+   fontSize: normalizeFontSize(10)
+ },
+ bottomBarLabel: {
+   fontFamily: fonts.inter[500],
+   fontSize: normalizeFontSize(10)
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->